### PR TITLE
AzureStack Keyvault Admin

### PR DESCRIPTION
**Pull request by deathly809**
Initial commit for the AzureStack KeyVault Admin Spec.

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by deathly809** [comment]
@jianghaolu There is some error with the node.js CI job itself I think.

```
npm ERR! code EINTEGRITY
npm ERR! sha512-CRxb97Ib3b6GWpF2bE37Vvqtd5KfsZ912OBWe0k92Zvk8LxY4WcQu4C6F4Numm78m8FdIWTAH4u9uT91riVdxQ== integrity checksum failed when using sha512: wanted sha512-CRxb97Ib3b6GWpF2bE37Vvqtd5KfsZ912OBWe0k92Zvk8LxY4WcQu4C6F4Numm78m8FdIWTAH4u9uT91riVdxQ== but got sha512-ouR0DO5JVkiOlGb9FenjY8svbvOTL0bkDnyJGcnZy9hrQxGjVzWO2Y3VOvQh8jCkAvBjaTKZMjWAcp6Fq1K2mA==. (2228224 bytes)
```

File: specification/azsadmin/resource-manager/keyvault/Microsoft.KeyVault.Admin/2017-02-01-preview/Quotas.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    },
+    "definitions": {
+        "Quota": {
+            "description": "KeyVault Quota.",
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Properties for keyvault quotas.",
+                    "value": null,
```
Comment (jianghaolu) [inline specification/azsadmin/resource-manager/keyvault/Microsoft.KeyVault.Admin/2017-02-01-preview/Quotas.json]: Syntax error: please remove "value".

File: specification/azsadmin/resource-manager/keyvault/Microsoft.KeyVault.Admin/2017-02-01-preview/KeyVault.json [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+                    "description": "Contains the localized display information for this particular operation / action.",
+                    "$ref": "#/definitions/Display"
+                }
+            }
+        },
+        "Display": {
+            "description": "Contains the localized display information for this particular operation / action.",
+            "properties": {
+                "provider": {
+                    "description": "The localized friendly form of the resource provider name � it is expected to also include the publisher/company responsible. It should use Title Casing and begin with \"Microsoft\" for 1st party services. e.g. \"Microsoft Monitoring Insights\" or \"Microsoft Compute.\"",
```
Comment (jianghaolu) [inline specification/azsadmin/resource-manager/keyvault/Microsoft.KeyVault.Admin/2017-02-01-preview/KeyVault.json]: Please fix these unknown characters: �

**Review by jianghaolu — CHANGES_REQUESTED** [review]

File: specification/azsadmin/resource-manager/keyvault/Microsoft.KeyVault.Admin/2017-02-01-preview/Quotas.json (hunk 1 above)
Comment (deathly809) [reply specification/azsadmin/resource-manager/keyvault/Microsoft.KeyVault.Admin/2017-02-01-preview/Quotas.json]: Done.  I don't get a syntax error locally, oav and autorest seem to work fine with it.

**Comment by deathly809** [comment]
@jianghaolu All checks passed!

**Review by jianghaolu — APPROVED** [review]

**Pull request merged**